Dart/src/com/jetbrains/lang/dart/DartBundle.properties

```diff
@@ -260,11 +260,11 @@ rename.need.reRun=Cannot rename.\nThere were changes in code after usages have b
 
 exportable.DartCodeFoldingSettings.presentable.name=Code Folding Settings
 
-analysis.server.status.good.text=Send feedback
+analysis.server.status.good.text=Dart Plugin: Submit Feedback
```

Comment:
`Analyzer: Submit Feedback`?

`Dart Analyzer: Submit Feedback`?

More descriptive is good; I think it would be good to keep it terse as well.